src/seedsigner/views/seed_views.py

```diff
@@ -6,14 +6,14 @@
 from embit import bip39
 from embit.descriptor import Descriptor
 from embit.networks import NETWORKS
-from typing import List
+from typing import List, Type
 
 from seedsigner.controller import Controller
 from seedsigner.gui.components import FontAwesomeIconConstants, SeedSignerCustomIconConstants
 from seedsigner.helpers import embit_utils
 from seedsigner.gui.screens import (RET_CODE__BACK_BUTTON, ButtonListScreen,
     WarningScreen, DireWarningScreen, seed_screens)
-from seedsigner.gui.screens.screen import LargeIconStatusScreen, LoadingScreenThread, QRDisplayScreen
+from seedsigner.gui.screens.screen import BaseScreen, LargeIconStatusScreen, LoadingScreenThread, QRDisplayScreen
```

Comment:
Do we really need to import BaseScreen and Type 7 lines above?

Reply:
Good catch! Those were added for a previous version but I missed them when changing and cleaning it up.